force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html

```diff
@@ -1,37 +1,33 @@
 <template>
-    <div class="slds-grid slds-wrap">
-        <div class="slds-col slds-text-heading_small">
-            <span>{componentTitle}</span>
+    <div class="slds-grid slds-grid_vertical-align-start">
+        <div class="slds-col slds-size--1-of-2 slds-text-title slds-text-title_bold slds-p-bottom_xx-large">
```

Comment:
I've been thinking about this styling. Let's try using `slds-text-heading_small` on this one so it stands out from the others. It took me a while to determine exactly where we wanted this to sit, but this is closer to the styling. We should be able to use this moving forward.